.github/workflows/build-package-docs.yaml

```diff
@@ -120,3 +136,69 @@ jobs:
         name: package-docs-build
         path: build-directory/docs/docsite/ansible-package-docs-html-*.tar.gz
         retention-days: 7
+
+  deploy-package-docs:
+    if: fromJSON(github.event.inputs.deploy)
+    needs: build-package-docs
+    runs-on: ubuntu-latest
+    environment:
+      name: deploy-package-docs
+      url: https://github.com/ansible/ansible-documentation
+    env:
+      dest-repo: ansible-community/package-doc-builds
```

Comment:
Why not put this in the ansible organization where the actual docs repo already lives?

Reply:
There was some discussion about this in one of the DaWGs meetings and the consensus seemed to be putting it in `ansible-community` would avoid any other access issues with the SC (although I know that is more about the source). I'm also aware that there is an ongoing conversation about splitting the source into "core docs" and "community docs", in which case "community docs" could move into `ansible-community`. Maybe @samccann can pipe in here with some more detail.

In essence the motivation to put the destination repo in `ansible-community` is to avoid having the entire publishing workflow be limited in terms of access to the community. I've wondered the same thing about putting the repo in `ansible` honestly so I'm glad you asked. We can still change where it goes. Do you have any preference for one or the other @gotmax23 ?

Reply:
I think we should keep it in the same organization as the source code to avoid confusion and allow us to use the same groups (namely @ansible/community-docs-maintainers) to manage perms.

Reply:
Maybe it seems premature to create the destination repo for the docs build in the `ansible-community` org. That whole "split core/community docs" conversation has been in the background along with the need to sort out SC permissions.

And it is true that different orgs will be problematic for using things like groups. I found out the hard way about the GitHub app I created (it couldn't do the content push across orgs).

At the same time I feel like the end goal here is creating a community-owned Read The Docs project for what is now the "package" docs. The `ansible-documentation` repo is tied to the "ansible-core" Read The Docs project and that should stay in the `ansible` org.

Perhaps we should use a repo in the `ansible` org now for consistency but it's something we can easily change if needed. I'd prefer not to hold up this PR on that point, if that is gating your approval @gotmax23 I'd really like to get this merged so we can have the package docs building on RTD and work out all the final details as we go.

Reply:
I don't feel super strongly and also don't want to hold this on what's ultimately an implementation detail.